runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesOverlay.java

```diff
@@ -103,7 +114,7 @@ public Dimension render(Graphics2D graphics)
 				renderProgressCircle(graphics, xpGlobe, startDrawX, DEFAULT_START_Y);
```

Comment:
Maybe have this return the circle and then use that to test mouseover instead of this drawnXpGlobes thing? You can check mouseover at render time as long as you dont draw the tooltip until after all of the globes are drawn.